components/public-api/gitpod/v1/workspace.proto

```diff
@@ -269,6 +590,22 @@ message WorkspacePhase {
   google.protobuf.Timestamp last_transition_time = 2;
 }
 
+enum WorkspaceScope {
```

Comment:
Is it really reusable between requests?  or it is better ro define nested scope for each message?